.travis.yml

```diff
@@ -1,28 +1,19 @@
 language: ruby
 rvm:
-  - 2.3
-  - 2.5
-  - 2.6
+  - 2.7
+  - 3.0
```

Comment:
Is this even still running on Travis? 😅 I wonder if wee need to move it over to using GHA? 😬

Reply:
Good question.

Reply:
Remotable has no tests that run after pushing a branch.  One note here is that the appraisal gem expects a travis yml file even locally.  So, that will have to stick around.  GHA is beyond my time block for this.  I only did it because I already had done some work on it.

Reply:
GHA.  Nailed it I think.